clients/src/main/java/org/apache/kafka/common/requests/FetchRequest.java

```diff
@@ -337,8 +362,27 @@ public AbstractResponse getErrorResponse(int throttleTimeMs, Throwable e) {
                 .setResponses(topicResponseList));
     }
 
+    public String clusterId() {
+        return data.clusterId();
+    }
+
+    public List<FetchTopic> topics() {
+        return data.topics();
+    }
+
+    public int maxWaitMs() {
+        return data.maxWaitMs();
+    }
```

Comment:
Do we use those ones anywhere?

Reply:
It is only used by KafkaRaftClient.

Reply:
The data object is public, so we may just be able to use that.

Reply:
Reverted.